Hello.java

```diff
@@ -1,9 +1,29 @@
 public class Hello {
 
-       Hello(){
-           System.out.print("Hello, World!");
+       private void PrintHello(String name){
+
+           if (name != null){
+               name = name.substring(0,1).toUpperCase() + name.substring(1);
```

Comment:
I ended up refactoring the code in a later issue so that should take care of this mistake.